lib/mrb/scripts/logger/flag.rb

```diff
@@ -0,0 +1,60 @@
+module Groonga
+  class Logger
+    class Flag
```

Comment:
In this API, this should be `Flags`. Because this object has multiple flag values. For example, `Flags.new(1 | 2 | 4)` is valid for `time`, `title` and `message` flags.

Reply:
I've got it.